sitewit/services.py

```diff
@@ -244,16 +244,30 @@ def get_campaign_subscription(self, account_token, campaign_id):
     def list_campaign_subscriptions(self, account_token):
         """Get all subscriptions to given campaign for given account.
 
-        Create subscription to a given Campaign for given Account.
-
         Args:
             account_token (str): account token.
-            campaign_id (str): campaign to subscribe.
         """
         return self.get(
             '/api/subscription/campaign/',
             headers=self._get_account_auth_header(account_token)).json()
 
+    def list_subscriptions(self, offset=0, limit=50):
```

Comment:
I don't think it's a good idea to mention the usage of certain method.   Method shouldn't know how it will be used, nope?